### PR TITLE
Libpcap version can be automatically generated based on the source.

### PR DESCRIPTION
**Pull request by hsluoyz**
This PR is to fix this issue: https://github.com/the-tcpdump-group/libpcap/issues/518
The corresponding change for Npcap is here: https://github.com/nmap/npcap/commit/db8b65da23a77c46624ea6278c15ade077a9e386

### CODE REVIEW

File: Win32/Prj/wpcap.vcxproj [hunk 1]
```diff
@@ -113,7 +113,8 @@
       <AdditionalDependencies>ws2_32.lib;..\..\..\..\packetWin7\Dll\Project\Release No NetMon and AirPcap\Packet.lib;%(AdditionalDependencies)</AdditionalDependencies>
     </Link>
     <PreBuildEvent>
-      <Command>win_flex -Ppcap_ -7 --outfile=..\..\scanner.c --header-file=..\..\scanner.h ..\..\scanner.l
+      <Command>call ..\..\..\GenVersion.bat
```
Comment (guyharris) [inline Win32/Prj/wpcap.vcxproj]: You need to add `GenVersion.bat` to the commit; `git add` it and then do `git commit --amend` and push the results up to GitHub (or whatever the equivalent is if you're not using the Git command-line tools).

Comment (hsluoyz) [inline Win32/Prj/wpcap.vcxproj]: OK. So I will move all the files into libpcap including the generated ``version.h''.

Comment (guyharris) [inline Win32/Prj/wpcap.vcxproj]: What do you mean by "move all the files into libpcap"?  The generated `version.h` shouldn't be checked into the Git repository, and the generated `version.c` shouldn't be, either.

All you need to do is add `GenVersion.bat` as a file in the top-level directory, along with this change.  You can update this commit to include `GenVersion.bat`, as well as the changes to the project file, the way I described.

Comment (hsluoyz) [inline Win32/Prj/wpcap.vcxproj]: By "all the files" I mean ``GenVersion.bat`` and ``version.h.in``. Currently they are both in Npcap.  The generated ``version.h`` is not checked in certainly.

**Pull request merged**